From 72f277f719824602d52bfbc93f998bb111c01da4 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Wed, 27 Mar 2024 11:51:48 +0100
Subject: [PATCH] ARM: configs: enable low power clocksource on
 fragment-02-multiv7_addons

Enable low power clocksource in STM32MP1 addons fragment.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: Ib95dbed3a95f07399874f68f58eac40db8d310cd
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/369765
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/fragment-02-multiv7_addons.config b/arch/arm/configs/fragment-02-multiv7_addons.config
index a0d67171494c..951115ff8480 100644
--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -7,6 +7,7 @@ CONFIG_BACKLIGHT_GPIO=y
 CONFIG_CAN_M_CAN=y
 CONFIG_CAN_M_CAN_PLATFORM=y
 CONFIG_CHR_DEV_SG=y
+CONFIG_CLKSRC_STM32_LP=y
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
 CONFIG_DEBUG_PREEMPT=y
 CONFIG_DEBUG_SECTION_MISMATCH=y
-- 
2.39.5

